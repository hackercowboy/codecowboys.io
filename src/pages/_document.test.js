import React from 'react';
import { shallow } from 'enzyme';

import Document from './_document.page';

jest.mock('next/document');

describe('<Document/>', () => {
  it('should render', () => {
    const wrapper = shallow(<Document />);
    expect(wrapper.shallow().debug()).toMatchSnapshot();
    expect(Document.getInitialProps()).toMatchSnapshot();
  });
});
