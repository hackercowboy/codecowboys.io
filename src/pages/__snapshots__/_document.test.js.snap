// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Document/> should render 1`] = `""`;

exports[`<Document/> should render 2`] = `Promise {}`;
